homeassistant/components/fmi/__init__.py

```diff
@@ -0,0 +1,369 @@
+"""The FMI (Finnish Meteorological Institute) component."""
+
+from datetime import date, timedelta
+import logging
+
+from dateutil import tz
+import fmi_weather_client as fmi
+from fmi_weather_client.errors import ClientError, ServerError
+import voluptuous as vol
+
+import homeassistant.components.sun as sun
+from homeassistant.const import CONF_LATITUDE, CONF_LONGITUDE, CONF_NAME, CONF_OFFSET
+import homeassistant.helpers.config_validation as cv
+from homeassistant.util import Throttle
+
+_LOGGER = logging.getLogger(__name__)
+
+CONF_MIN_HUMIDITY = "min_relative_humidity"
+CONF_MAX_HUMIDITY = "max_relative_humidity"
```

Comment:
A lot of integrations put these constants in a const module

Reply:
I have moved them to a separate file as well.